src/lib/validations/sql.js

```diff
@@ -247,24 +247,29 @@ const perLineValidations = ( line: string ) => {
 	lineNum += 1;
 };
 
+const execute = ( line: string, isImport: boolean = true ) => {
+	perLineValidations( line, isImport );
+};
+
+const postLineExecutionProcessing = async ( { fileName, isImport }: PostLineExecutionProcessingParams ) => {
+	await postValidation( fileName, isImport );
+};
+
 export const staticSqlValidations = {
-	execute: ( line: string ) => {
-		perLineValidations( line );
-	},
-	postLineExecutionProcessing: async ( { fileName, isImport }: PostLineExecutionProcessingParams ) => {
-		await postValidation( fileName, isImport );
-	},
+	execute,
+	postLineExecutionProcessing,
```

Comment:
same as above, can we just change this to:
```suggestion
	postLineExecutionProcessing: postValidation,
```

...?

Reply:
Fixed!